parent/pom.xml

```diff
@@ -773,6 +773,13 @@
 
 		<profile>
 			<id>deploy-to-jboss.org</id>
+			<properties>
+				<deploySrc>${project.build.directory}/repository</deploySrc>
+				<deployDestServer>tools@filemgmt.jboss.org:downloads_htdocs/jbosstools</deployDestServer>
```

Comment:
Since URL is only used for BUILD_DESCRIPTION, and nesting this in a mojo means that BUILD_DESCRIPTION is never written to the console, and therefore not picked up by Jenkins to turn that into a Build Description, we can probably drop that field.

Also, since rsync.sh uses "source" and "target" (because I like the parallel nature of two 6-letter words), maybe we should use these three?

* deployDestination (could be a local path, doesn't have to be user@server:/path), 
* deploySourceFolder
* deployTargetFolder

We could merge the destination path and target folder into one variable IFF we could guarantee that the intermediate folders in the tree were already created. But since we can't, the only way I have to guarantee that is to manually create them over sftp:

https://github.com/jbosstools/jbosstools-build-ci/blob/jbosstools-4.3.x/publish/rsync.sh#L58

Can you think of a work around for this?